common/anything-sync-daemon.in

```diff
@@ -14,9 +14,16 @@ VERS="@VERSION@"
 
 ASDCONF=${ASDCONF:-"/etc/asd.conf"}
 DAEMON_FILE=${DAEMON_FILE:-"/run/asd"}
+if [[ "$ASDCONF" != "/etc/asd.conf" ]]; then
+    ASDNAME="${ASDCONF##*/}"
+    ASDNAME="-${ASDNAME%.conf}"
```

Comment:
```suggestion
    ASDNAME="${ASDNAME%.conf}"
```
I would prefer that the `ASDNAME` variable doesn't have prefix `-` in it. And the variables using it add that prefix in them. Like `TMP="$VOLATILE/asd-$ASDNAME-$USER$DIR"`